dtfs-central-api/src/v1/controllers/utilisation-report-service/post-report-data-validation.controller/index.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiError, CustomExpressRequest } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import { HttpStatusCode } from 'axios';
+import { validateUtilisationReportCsvData } from '../../../../services/utilisation-report-data-validator';
+import { PostReportDataValidationPayload } from '../../../routes/middleware/payload-validation/validate-post-validate-utilisation-report-data-payload';
+
+export type PostReportDataValidationRequest = CustomExpressRequest<{
+  reqBody: PostReportDataValidationPayload;
+}>;
+
+/**
+ * Controller to handle post request to report data validation route
+ * @param req - The request
+ * @param res - The response
+ */
+export const postReportDataValidation = (req: PostReportDataValidationRequest, res: Response) => {
+  const { reportData } = req.body;
+
+  try {
+    const csvValidationErrors = validateUtilisationReportCsvData(reportData);
+    return res.status(HttpStatusCode.Ok).send({ csvValidationErrors });
+  } catch (error) {
+    const errorMessage = 'Failed to validate report data';
+    console.error(errorMessage, error);
```

Comment:
@abhi-markan should this be %s %s?

Reply:
There are currently 34 instances of this throughout the codebase. If it needs to change, I suggest we do it all in one go.